cpp/util/fake_etcd.cc

```diff
@@ -25,6 +25,35 @@ using util::StatusOr;
 using util::Task;
 
 namespace cert_trans {
+namespace {
+
+
+string NormalizeKey(const string& input) {
```

Comment:
Good catch. It does handle ```.``` and ```..``` properly, it looks like! I'll revise this accordingly...